docs/migration/uwp-projects.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Xamarin.UWP project migration to .NET MAUI WinUI"
+description: "Learn how to migrate a Xamarin.UWP project to a .NET WinUI project."
+ms.date: 11/13/2023
+content_well_notification:
+  - AI-contribution
+---
+
+# Xamarin.UWP project migration to .NET MAUI WinUI
+
+## Prerequisites
+
+- Visual Studio 2023 Preview or later with and ensure that .NET Multi-platform App UI development is selected.
+- The latest version of the Windows App SDK extension for Visual Studio.
+- A Xamarin.Forms UWP app that targets Windows 10 version 1809 or later.
+
+## Project Changes
+
+A .NET 8 project for a .NET MAUI WinUI app is similar to the following example:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType> <!-- in Xamarin.Forms this was AppContainerExe -->
+    <UseWinUI>true</UseWinUI>
+    <UseMaui>true</UseMaui>
+    <EnableMsixTooling>true</EnableMsixTooling>
+    <Platforms>x86;x64;ARM64</Platforms>
+  </PropertyGroup>
+</Project>
+```
+
+The `<OutputType>` property was previously AppContainerExe in Xamarin.UWP propjects. For a library project, omit the `$(OutputType)` property completely or specify `Library` as the property value.
+
+## Changes to MSBuild properties
+
+Platform properties should be replaced in favor of the following .NET runtime identifiers:
+
+```xml
+<PropertyGroup>
+  <!-- Used in .NET MAUI WinUI projects -->
+   <Platforms>x86;x64;ARM64</Platforms>
+</PropertyGroup>
+```
+
+For more information about runtime identifiers, see [.NET RID Catalog](/dotnet/core/rid-catalog).
+
+## API Changes
+
+To safely use recent or older APIs, you can declare a `SupportedOSPlatformVersion` in your project or use the <xref:System.OperatingSystem.IsWindowsVersionAtLeast%2A> API at runtime:
```

Comment:
I think we need to show how to declare a `SupportedOSPlatformVersion`, or link to an existing doc that shows this.